terraform/locals.tf

```diff
@@ -1,80 +0,0 @@
-locals {
```

Comment:
This file should not have been deleted!  We need it. :skull_and_crossbones: Here be :dragon:!